modules/core/src/main/java/org/apache/ignite/internal/IgnitionEx.java

```diff
@@ -1928,6 +1931,23 @@ private void start0(GridStartContext startCtx, IgniteConfiguration cfg, TimeBag
                     GridIoPolicy.IDX_POOL,
                     oomeHnd
                 );
+
+                int buildIdxThreadPoolSize = cfg.getBuildIndexThreadPoolSize();
+
+                validateThreadPoolSize(buildIdxThreadPoolSize, "build-idx");
+
+                buildIdxExecSvc = new IgniteThreadPoolExecutor(
+                    "build-idx-runner",
+                    cfg.getIgniteInstanceName(),
+                    0,
+                    buildIdxThreadPoolSize,
+                    DFLT_THREAD_KEEP_ALIVE_TIME,
```

Comment:
Default keepalive time is 60sec. 
Is there any reason to have idle threads alive?

Reply:
I can't find the reason, but what's wrong with the default value?